lib/structures/Subscription.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const Base = require("./Base.js");
+
+/**
+ * Represents a subscription
+ * @prop {Number} currentPeriodEnd The end date of the current subscription period
+ * @prop {Number} currentPeriodStart The start date of the current subscription period
+ * @prop {Number} canceledAt When the subscription was canceled
```

Comment:
These are possibly `null`
```suggestion
 * @prop {Number?} canceledAt When the subscription was canceled
 * @prop {Number?} currentPeriodEnd The end date of the current subscription period
 * @prop {Number?} currentPeriodStart The start date of the current subscription period
```